src/imagetextalternative.js

```diff
@@ -98,10 +95,14 @@ export default class ImageTextAlternative extends Plugin {
 	 */
 	_createBalloonPanel() {
 		const editor = this.editor;
+		const command = editor.commands.get( 'imageTextAlternative' );
 
 		const panel = new ImageBalloonPanel( editor );
 		const form = new TextAlternativeFormView( editor.locale );
 
+		form.labeledInput.inputView.bind( 'isReadOnly' ).to( command, 'isEnabled', value => !value );
+		form.saveButtonView.bind( 'isEnabled' ).to( command );
```

Comment:
This way the form will be re–usable with all its features. In your implementation, if you want to re–use the form, you got to repeat the code from the lines above each time.